README.md

```diff
@@ -37,25 +38,16 @@ The JWKMS can be deploy in two different ways:
 
 ## Centralised JWKMS instance
 
-The JWKMS can be deploy as a single dedicated instance. In that configuration, the JWKMS can manage keys for multiple
- applications at the same time. This means that your other micro-services can delegate the JWTs manipulation and JWK management
- to the JWKMS.
+The JWKMS can be deployed as a single dedicated micro-service instance. In that configuration, the JWKMS can be used to manage keys for multiple applications at the same time. This means that your other micro-services can delegate all JWTs manipulation and JWK management to the JWKMS.
```

Comment:
single dedicated micro-service instance. should just be can be deployed as a microservice instance as it takes away the fact it can horizontally scale